docs/docs/advanced/modules.md

```diff
@@ -94,9 +94,9 @@ $ trivy module uninstall ghcr.io/aquasecurity/trivy-module-spring4shell
 ```
 
 ## Building Modules
-It supports TinyGo only at the moment.
+It supports Go only at the moment.
 
-### TinyGo
+### Go
```

Comment:
update `PostScan(serialize.Results) (serialize.Results, error)` to `PostScan(results types.Results) (types.Results, error)` bellow.

Reply:
https://github.com/aquasecurity/trivy/blob/0d50527e5f53f13c91de25c9c74f589e1731b5ad/docs/docs/advanced/modules.md?plain=1#L116

https://github.com/aquasecurity/trivy/blob/0d50527e5f53f13c91de25c9c74f589e1731b5ad/docs/docs/advanced/modules.md?plain=1#L239

https://github.com/aquasecurity/trivy/blob/0d50527e5f53f13c91de25c9c74f589e1731b5ad/docs/docs/advanced/modules.md?plain=1#L249

Reply:
Thanks for catching them.
https://github.com/aquasecurity/trivy/pull/8496/commits/4ff9d718c192856641c196e6f88e6b505954ff2f